xla/hlo/transforms/collectives/all_gather_cse.h

```diff
@@ -0,0 +1,59 @@
+/* Copyright 2024 The OpenXLA Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef ALL_GATHER_CSE_H
+#define ALL_GATHER_CSE_H
```

Comment:
Should be XLA_HLO_TRANSFORMS_COLLECTIVES_ALL_GATHER_CSE_H

Reply:
Good catch

Reply:
Fixed. @mooskagh can you approve now?